devfiles/java-openliberty/devfile.yaml

```diff
@@ -1,40 +1,53 @@
-schemaVersion: 2.0.0
+apiVersion: 1.0.0
```

Comment:
This devfile is in devfile 1.0 format instead of devfile 2.0 which this repo is for.

Reply:
ok, resolved, i copied this source originally from the wrong branch in our release repo